activerecord/lib/active_record/associations.rb

```diff
@@ -300,10 +300,10 @@ def association_instance_set(name, association)
     #
     # === A word of warning
     #
-    # Don't create associations that have the same name as instance methods of
-    # ActiveRecord::Base. Since the association adds a method with that name to
-    # its model, it will override the inherited method and break things.
-    # For instance, +attributes+ and +connection+ would be bad choices for association names.
+    # Don't create associations that have the same name as [instance methods](http://apidock.com/rails/ActiveRecord/Base) of
```

Comment:
I dug into that link and found this:

http://api.rubyonrails.org/classes/ActiveRecord/Core.html#method-c-new, which pops the user to a list of public instance methods. Is this link sufficient?